### ISSUE TITLE
Flexing OptionsMenu

### ISSUE THREAD

**Issue by Stanley521** [izzisolomon/react-native-options-menu#7]
unable to flex OptionsMenu so that it fills the parent height.

Edit: 
for example this,

<View style={{ flex: 1, backgroundColor: 'pink', flexDirection: 'row', justifyContent: 'center', alignItems: 'center'}}> <OptionsMenu customButton={( <View style={{ backgroundColor: 'green'}}> <Text style={{color: 'white'}}> More </Text> </View> )} destructiveIndex={1} options={["Test", "Cancel"]} actions={[ () => {}]} /> </View>

I would like the OptionsMenu to flex and fill the view.
How can I accomplish it?

I would like the green(Touchable) to flex to pink.
Do you know how can I accomplish it?

**Comment by hali241997** [comment]
Can you explain the issue?

**Comment by Stanley521** [comment]
for example this,
`
<View style={{ flex: 1, backgroundColor: 'pink', flexDirection: 'row', justifyContent: 'center', alignItems: 'center'}}>
      <OptionsMenu
      customButton={(
        <View style={{ backgroundColor: 'green'}}>
          <Text style={{color: 'white'}}> More </Text>
        </View>
      )}
      destructiveIndex={1}
      options={["Test", "Cancel"]}
      actions={[ () => {}]}
      />
</View>
`
I would like the OptionsMenu to flex and fill the view.
How can I accomplish it?

**Comment by hali241997** [comment]
![Screenshot_20200123_132948_com test](https://user-images.githubusercontent.com/33332648/72968122-a5b28780-3de4-11ea-850d-ab0749502b13.jpg)

Do you want something like this?

**Comment by Stanley521** [comment]
correct, sorry for the late reply! I have updated the code, I would like the green(Touchable) to flex to pink.
Do you know how can I accomplish it?
